client/src/Service/Client/Internal/StatsApi.php

```diff
@@ -9,6 +9,7 @@
 class StatsApi
```

Comment:
Few formatting issues in this comment with some `span` not lining up with their closing counterparts. While this is purely a alignment issue it helps easily identify where HTML start and end when you don't have a fancy IDE/Editor that highlights them for you :)